addons_rksv/pos_rksv/views/pos_config.xml

```diff
@@ -16,7 +16,8 @@
                     </header>
                 </xpath>
                 <field name="iface_cashdrawer" position="after">
-                    <field name="iface_rksv" />
+                    <field name="rksv_at" invisible="True"/>
+                    <field name="iface_rksv" attrs="{'invisible': [('rksv_at', '=', False)]}"/>
```

Comment:
You misunderstood...what if it was activated before and later the rksv_at is disabled...then you have an activated iface_rksv which can't be changed back...so there must be an onchange somewhere.